R/pkg/R/mllib.R

```diff
@@ -99,10 +114,8 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
             return(new("GeneralizedLinearRegressionModel", jobj = jobj))
           })
 
-#' Fits a generalized linear model (R-compliant).
-#'
-#' Fits a generalized linear model, similarly to R's glm().
-#'
+#' @title Generalized Linear Models (R-compliant)
```

Comment:
@junyangq Could you remove `@title`? We might need an empty line after this line. See attached screenshot:

![screen shot 2016-06-21 at 8 26 07 pm](https://cloud.githubusercontent.com/assets/829644/16253802/7870fdf6-37ee-11e6-97e8-98bf5b16714a.png)